src/app/core/export/data-transformation-service/data-transformation.service.spec.ts

```diff
@@ -264,68 +267,81 @@ describe("DataTransformationService", () => {
     todayNote.date = new Date();
     await entityMapper.save(todayNote);
 
+    const childrenService = TestBed.inject(ChildrenService);
+    const getNotesInTimespan = spyOn(childrenService, "getNotesInTimespan");
+
+    getNotesInTimespan.and.resolveTo([yesterdayNote, todayNote]);
+    const startDate = moment().subtract(5, "days").toDate();
     let result = await service.queryAndTransformData(
       [
         {
           query: `${Note.ENTITY_TYPE}:toArray`,
           subQueries: [{ query: "subject" }],
         },
       ],
-      moment().subtract(5, "days").toDate(),
+      startDate,
     );
 
     expect(result.map((x) => x.subject)).toEqual(["yesterday", "today"]);
+    expect(getNotesInTimespan).toHaveBeenCalledWith(
```

Comment:
This is somehow flaky, failing sometimes

> DataTransformationService > should support time spans for data export
Expected spy getNotesInTimespan to have been called with:
  [ Date(Thu Jul 10 2025 17:10:16 GMT+0200 (Central European Summer Time)), <jasmine.anything> ]
but it was never called.